File: Python-projects/Data Structures and Algorithms/Programs/Week2_Assignment/Matched.js
let matched = (s) => {
    let c = s.split("");
    let i = 0;
    let count = 0;
    while (count >= 0 && i < c.length) {
        if (c[i] == "(") {
            count++;
        }
        if (c[i] == ")") {
            count--;
        }
        i++;
    }
    return count == 0
}

console.log(matched("()()()()"));

File: Python-projects/Data Structures and Algorithms/Programs/Week2_Assignment/matched.py
def matched(s):
    count =0;
    i=0;
    while(i<len(s) and count>=0):
      if(s[i]=="("):
         count= count+1
      if(s[i]==")"):
         count= count-1
      i=i+1
    return count==0;        

print(matched("(((()))"))    

           

#
# make list of brackets
# intitialise count =0

#if while count=0 and c[idx] == ''
# 
#  
# #